server/game/PropertyTypes/BlankingProperty.js

```diff
@@ -0,0 +1,19 @@
+class BlankingProperty {
+    constructor() {
+        this.blanks = {};
+    }
+
+    addBlankingSource(source, includingTraits = false) {
+        this.blanks[source.uuid] = { source: source, includingTraits: includingTraits };
+    }
+
+    removeBlankingSource(source) {
+        delete this.blanks[source.uuid];
+    }
+
+    isBlank(includingTraits = false) {
```

Comment:
Could we split this into two separate methods (e.g. `isTextBlank()` and `areTraitsBlank()`)? Passing boolean params to a getter is hard to understand unless you jump back and forth between usage and definition. I did a double take when I saw `isBlank(true)`.